generator/api/src/main/java/io/fabric8/maven/generator/api/FromSelector.java

```diff
@@ -126,7 +129,13 @@ public boolean isJava11() {
     }
 
     private boolean isEleven(String s) {
-        return Configs.asInt(s) == 11;
+        boolean result = false;
+        try {
+            result = Configs.asInt(s) == 11;
+        } catch (NumberFormatException e) {
+            return false;
```

Comment:
This is redundant (result is already false)

Reply:
Check now :)